### ISSUE TITLE
app-accessibility/gespeaker: No supported implementation in PYTHON_COMPAT.

### ISSUE THREAD

**Issue by EmRowlands** [vaeth/mv-overlay#21]
Similar to #20, package relies on python 2.7, causing errors in the global scope. There have also been no commits upstream since November 2020.

The error appears even if the package is not installed:
```
$ eclean -p distfiles
 * Building file list for distfiles cleaning...
 * ERROR: app-accessibility/gespeaker-0.8.6-r2::mv failed (depend phase):
 *   No supported implementation in PYTHON_COMPAT.
 *
 * Call stack:
 *                   ebuild.sh, line 609:  Called source '/var/db/repos/mv/app-accessibility/gespeaker/gespeaker-0.8.6-r2.ebuild'
 *   gespeaker-0.8.6-r2.ebuild, line   7:  Called inherit 'distutils-r1'
 *                   ebuild.sh, line 314:  Called __qa_source '/var/db/repos/gentoo/eclass/distutils-r1.eclass'
 *                   ebuild.sh, line 112:  Called source '/var/db/repos/gentoo/eclass/distutils-r1.eclass'
 *         distutils-r1.eclass, line 105:  Called inherit 'python-r1'
 *                   ebuild.sh, line 314:  Called __qa_source '/var/db/repos/gentoo/eclass/python-r1.eclass'
 *                   ebuild.sh, line 112:  Called source '/var/db/repos/gentoo/eclass/python-r1.eclass'
 *            python-r1.eclass, line 260:  Called _python_set_globals
 *            python-r1.eclass, line 199:  Called _python_set_impls
 *      python-utils-r1.eclass, line 156:  Called die
 * The specific snippet of code:
 *                      die "No supported implementation in PYTHON_COMPAT."
 *
 * If you need support, post the output of `emerge --info '=app-accessibility/gespeaker-0.8.6-r2::mv'`,
 * the complete build log and the output of `emerge -pqv '=app-accessibility/gespeaker-0.8.6-r2::mv'`.
 * Working directory: '/usr/lib/python3.8/site-packages'
 * S: '/var/tmp/portage/app-accessibility/gespeaker-0.8.6-r2/work/gespeaker-0.8.6'
 * ERROR: app-accessibility/gespeaker-0.8.6_p0::mv failed (depend phase):
 *   No supported implementation in PYTHON_COMPAT.
 *
 * Call stack:
 *                   ebuild.sh, line 609:  Called source '/var/db/repos/mv/app-accessibility/gespeaker/gespeaker-0.8.6_p0.ebuild'
 *   gespeaker-0.8.6_p0.ebuild, line   7:  Called inherit 'distutils-r1'
 *                   ebuild.sh, line 314:  Called __qa_source '/var/db/repos/gentoo/eclass/distutils-r1.eclass'
 *                   ebuild.sh, line 112:  Called source '/var/db/repos/gentoo/eclass/distutils-r1.eclass'
 *         distutils-r1.eclass, line 105:  Called inherit 'python-r1'
 *                   ebuild.sh, line 314:  Called __qa_source '/var/db/repos/gentoo/eclass/python-r1.eclass'
 *                   ebuild.sh, line 112:  Called source '/var/db/repos/gentoo/eclass/python-r1.eclass'
 *            python-r1.eclass, line 260:  Called _python_set_globals
 *            python-r1.eclass, line 199:  Called _python_set_impls
 *      python-utils-r1.eclass, line 156:  Called die
 * The specific snippet of code:
 *                      die "No supported implementation in PYTHON_COMPAT."
 *
 * If you need support, post the output of `emerge --info '=app-accessibility/gespeaker-0.8.6_p0::mv'`,
 * the complete build log and the output of `emerge -pqv '=app-accessibility/gespeaker-0.8.6_p0::mv'`.
 * Working directory: '/usr/lib/python3.8/site-packages'
 * S: '/var/tmp/portage/app-accessibility/gespeaker-0.8.6_p0/work/gespeaker-0.8.6_p0'
```

**Comment by vaeth** [comment]
SImilarly for sdlmame...
Fixed both in the same way: Claiming (without testing) that current python is supported. Probably this will not work, and I might have to remove the package if I do not find an easy solution (I am not willing to spend much time).
Anyway, marking as fixed for the moment.

**Comment by EmRowlands** [comment]
Upstream's README says:
```
The following dependencies are needed to let Gespeaker work:
 - python2: all the whole code is written for Python 2.x
```
As well as gespeaker.py explicitly stating `#!/usr/bin/env python2`.

Given the contents of this[1] issue, I doubt that python 3 support is coming any time soon.

 [1]: https://github.com/muflone/gespeaker/issues/69
